.claude/commands/kb-board.md

```
@@ -0,0 +1,44 @@
+---
+allowed-tools:
+  - mcp__gitkb__kb_list
+  - mcp__gitkb__kb_show
+  - mcp__gitkb__kb_graph
+  - Bash(git kb:*)
+description: Show GitKB kanban board with task status columns
+---
+
+Display the kanban board and provide actionable context about the current workstream.
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)